bindings/rust/aws-kms-tls-auth/DEVELOPMENT.md

```diff
@@ -0,0 +1,49 @@
+# Managing Algorithm & Format Changes.
+
+## Backwards Compatibility: Required
+Changes must always be backwards compatible. More specifically, server's must always be able to deserialize earlier versions of PskIdentities, otherwise all of the in-flight communications would fail when an upgrade happens
```

Comment:
```suggestion
Changes must always be backwards compatible. More specifically, server's must always be able to deserialize earlier versions of PskIdentities, otherwise all of the in-flight communications would fail when an upgrade happens.
```